ui/helpers/utils/permission.js

```diff
@@ -72,8 +72,7 @@ function getSnapNameComponent(snapName) {
 export const PERMISSION_DESCRIPTIONS = deepFreeze({
   [RestrictedMethods.eth_accounts]: ({ t }) => ({
     label: t('permission_ethereumAccounts'),
-    leftIcon: getLeftIcon(IconName.Eye),
```

Comment:
Curious about this change 🤔

Reply:
Current state of this PR when using that permission:
![Screenshot 2024-03-19 at 17 02 14](https://github.com/MetaMask/metamask-extension/assets/13301024/67dbb1fb-9214-4b0b-9503-85b1b250b9ee)

Reply:
Some of the Snaps permissions are using `getLeftIcon`, but they probably should not, it seems to be inherited legacy 🤔

Reply:
Function `getLeftIcon` alongside its usage is removed from everywhere now, considering it's redundant.